flink-tests/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTriggerSavepointITCase.java

```diff
@@ -110,11 +111,8 @@ private void setUpWithCheckpointInterval(long checkpointInterval) throws Excepti
                                 0),
                         null);
 
-        jobGraph =
-                JobGraphBuilder.newStreamingJobGraphBuilder()
-                        .addJobVertex(vertex)
-                        .setJobCheckpointingSettings(jobCheckpointingSettings)
-                        .build();
+        jobGraph = JobGraphTestUtils.streamingJobGraph(vertex);
+        jobGraph.setSnapshotSettings(jobCheckpointingSettings);
```

Comment:
I'm not sold on these changes; they revert us back to a pattern where we explicitly mutate the JobGraph and have to use badly-named APIs.